prov/efa/src/rxr/rxr_msg.c

```diff
@@ -212,6 +212,17 @@ ssize_t rxr_msg_post_rtm(struct rxr_ep *rxr_ep, struct rxr_tx_entry *tx_entry)
 	if (tx_entry->total_len <= max_rtm_data_size) {
 		ctrl_type = (delivery_complete_requested) ?
 			RXR_DC_EAGER_MSGRTM_PKT : RXR_EAGER_MSGRTM_PKT;
+
+		if (delivery_complete_requested &&
+		    !(peer->flags & RXR_PEER_HANDSHAKE_RECEIVED)) {
+			tx_entry->state = RXR_TX_QUEUED_CTRL;
```

Comment:
The next 4 lines appear repeated. Can we make them an inline function.

Reply:
Where shall I put it. Can I put that inline function in `rxr_pkt_cmd.h`

Reply:
That is OK.

Reply:
queueing logic no longer used. resolving.